rswag-specs/lib/tasks/rswag-specs_tasks.rake

```diff
@@ -16,11 +16,13 @@ namespace :rswag do
         ''
       )
 
+      t.rspec_opts = [additional_rspec_opts]
+
       if Rswag::Specs::RSPEC_VERSION > 2 && Rswag::Specs.config.swagger_dry_run
-        t.rspec_opts = ['--format Rswag::Specs::SwaggerFormatter', '--dry-run', '--order defined'] << additional_rspec_opts
+        t.rspec_opts += ['--format Rswag::Specs::SwaggerFormatter', '--dry-run', '--order defined']
       else
         ActiveSupport::Deprecation.warn('Rswag::Specs: WARNING: Support for RSpec 2.X will be dropped in v3.0')
```

Comment:
It's separate from this PR, but shouldn't this warning only print if `RSPEC_VERSION <= 2`? It should be independent of `swagger_dry_run`, right?